File: CS/CustomResourceHeaderExample/ViewModels/MainViewModel.cs
using CustomResourceHeaderExample.Data;
using DevExpress.Mvvm;
using System.Collections.ObjectModel;

namespace CustomResourceHeaderExample.ViewModels {
    public class MainViewModel : ViewModelBase {

        public MainViewModel() {
            MainResources = ResourceData.CreateResources();
        }
        public ObservableCollection<MyResource> MainResources { get; }
    }
}

File: VB/CustomResourceHeaderExample/ViewModels/MainViewModel.vb
Imports CustomResourceHeaderExample.Data
Imports DevExpress.Mvvm
Imports System.Collections.ObjectModel

Namespace CustomResourceHeaderExample.ViewModels

    Public Class MainViewModel
        Inherits ViewModelBase

        Public Sub New()
            MainResources = ResourceData.CreateResources()
        End Sub

        Public ReadOnly Property MainResources As ObservableCollection(Of MyResource)
    End Class
End Namespace
